spec/pg_ha_migrations_spec.rb

```diff
@@ -5,6 +5,30 @@
     expect(ActiveRecord::Migration.disable_ddl_transaction).to be_truthy
   end
 
+  describe "config" do
+    before(:each) do
+      PgHaMigrations.instance_variable_set(:@config, nil)
+    end
+
+    context "disable_default_migration_methods" do
+      subject(:value) { PgHaMigrations.config.disable_default_migration_methods }
```

Comment:
We'd prefer to have these two tests assert directly on the config method instead of pulling it out into a `subject` (this is a bit of house style request again 😄).